plugin.rb

```diff
@@ -65,4 +65,12 @@ module ::DiscourseAi
   end
 
   reloadable_patch { |plugin| Guardian.prepend DiscourseAi::GuardianExtensions }
+
+  register_modifier(:post_with_secure_uploads?) do |_, _, topic|
+    if SharedAiConversation.exists?(target: topic)
```

Comment:
I think to save querying only do this check on PMs?

Reply:
Yes definitely makes sense :+1: